src/odemis/driver/test/tescan_test.py

```diff
@@ -189,6 +372,8 @@ def test_probe_current(self):
         time.sleep(6)  # Wait for value refresh
         self.assertAlmostEqual(orig_probe_current, ebeam.probeCurrent.value)
 
+    # skip this method if running in a simulator not specifically for
+    # the Mira apparently the wait time of 6s is quite critical here
```

Comment:
shouldn't you then add a `@unittest.skip`?

Reply:
I dont know what changed but now this test is passing constantly so I removed the whole comment eventually.